force-app/main/custom-metadata-saver/classes/CustomMetadataTableController.cls

```diff
@@ -0,0 +1,128 @@
+//----------------------------------------------------------------------------------------------------//
+// This file is part of the Custom Metadata Saver project, released under the MIT License.            //
+// See LICENSE file or go to https://github.com/jongpie/CustomMetadataSaver for full license details. //
+//----------------------------------------------------------------------------------------------------//
+
+public without sharing class CustomMetadataTableController {
+    private static String BASE_URL = System.Url.getOrgDomainUrl().toExternalForm() + '/services/data/v49.0';
```

Comment:
you've got my attention with that API version - any particular reason for `49` ?

Reply:
I think this is a result of some good ole copypasta - I copied part of this from my ApexCallouts repo & forgot to update the API version, I'll update it!

Reply:
Fixed! I've bumped it to `v52.0` instead of `v49.0`